docs/introduction.md

```diff
@@ -161,7 +161,7 @@ to install SAT as a separate product stream. Any version of SAT installed as a s
   only available with the full SAT product stream.
 
 If the SAT product stream is not installed, there will be no configuration content for SAT in VCS. Therefore, CFS
-configurations that apply to NCNs (for example, `ncn-personalization`) should not include a SAT layer.
+configurations that apply to management NCNs (for example, `management-22.03`) should not include a SAT layer.
```

Comment:
Not sure if you meant 22.03 or 23.03 here, but probably you meant 23.03 here as well.

Reply:
Yep, sure did, thanks!